database/sampleDataMethods.js

```diff
@@ -19,21 +26,30 @@ module.exports.type = type;
 /* =================== Page Number ===================== */
 
 const pageNum = () => {
-  return faker.random.number({'min': 250, 'max': 800});
+  return getRandomInt(350, 450);
 };
 module.exports.pageNum = pageNum;
 
 /* =================== Publisher ===================== */
 
 const publisher = () => {
-  return faker.company.companyName();
+  let arr = ['Swaniawski Inc',
```

Comment:
Rather than having variable names describe the data structure (eg, `arr`), what do you think about having it describe what it holds (eg, `publishers`)?

Reply:
Thanks for all your comments.